src/common/status.h

```diff
@@ -397,3 +397,16 @@ bool StatusIsOK(const T& v) {
     if (!StatusIsOK(status)) return std::forward<decltype(status)>(status); \
     std::forward<decltype(status)>(status);                                 \
   }))
+
+// NOLINTNEXTLINE
+#define RETURN_IF_ERROR(...)                                                \
```

Comment:
Hmmm I don't think you've got my point. As I've said, this macro is useless and can be replaced by GET_OR_RET. Actually it's function is a subset of GET_OR_RET.

Reply:
> Hmmm I don't think you've got my point. As I've said, this macro is useless and can be replaced by GET_OR_RET. Actually it's function is a subset of GET_OR_RET.

Firstly, Thank you very much for reviewing my submitted PR and providing valuable feedback! I truly appreciate your input and am grateful for the time you’ve spent helping me improve the code.


This macro can indeed be replaced by `GET_OR_RET`.